x/concentrated-liquidity/export_test.go

```diff
@@ -284,8 +284,8 @@ func UpdateAccumAndClaimRewards(accum accum.AccumulatorObject, positionKey strin
 	return updateAccumAndClaimRewards(accum, positionKey, growthOutside)
 }
 
-func (k Keeper) ClaimAllIncentivesForPosition(ctx sdk.Context, positionId uint64) (sdk.Coins, sdk.DecCoins, error) {
-	return k.claimAllIncentivesForPosition(ctx, positionId)
+func (k Keeper) PrepareClaimAllIncentivesForPosition(ctx sdk.Context, positionId uint64) (sdk.Coins, sdk.Coins, error) {
+	return k.prepareClaimAllIncentivesForPosition(ctx, positionId)
```

Comment:
Note to reviewer: renamed `claimAllIncentivesForPosition` to `prepareClaimAllIncentivesForPosition` and moved the community pool send to `collectIncentives`. This way both bank sends happen in the same function.